src/spatialdata_plot/pl/utils.py

```diff
@@ -490,11 +490,8 @@ def _prepare_cmap_norm(
     cmap: Colormap | str | None = None,
     norm: Normalize | None = None,
     na_color: ColorLike | None = None,
-    vmin: float | None = None,
```

Comment:
@timtreis I don't remember the outcome of the discussion with the user that reported this. Is this the way to go (=letting users only use `norm` and not `vmin`, `vmax`) or shall we remove `vcenter` only and keep `vmin`, `vmax` and use them to initialize the default `Normalize` object?

Reply:
In the discussion it was stated that vmin and vmax are removed. This function is only internally called

Reply:
So we agree on clip 'True' by default if user does not provide normaloze object?

Reply:
If `vmin` and `vmax` are not exposed to the user (and hence they are `None`), then `clip` will have no effect because when exposed `vmin`, `vmax` are `None`, the data limits are used. So I would keep the default `clip` to be `False` (which is `matplotlib`'s default).

Reply:
One thing, if `vmin`, `vmax` are removed from `pl.render_shapes()`, we should throw an informative exception or deprecation warning, explaining to the user that `norm` should be used instead. Could you add that please?

Reply:
They have not been removed in this PR though and the public functions thus already did not contain these parameters

Reply:
I did add a deprecation warning in case of the arguments being passed as kwargs